syntheticsclientv2/integration_test.go

```diff
@@ -245,12 +245,13 @@ func CreateHttpCheckV2(test string, c *Client) (int, error) {
 	}
 	fmt.Printf("\nReq was: \n%v\n", reqDetail)
 	JsonPrint(res)
+	inputHttpCheckV2Data = HttpCheckV2Input{}
```

Comment:
@greatestusername-splunk  not sure if this would be the best way to do this, but with this, the validations seem to be reset as expected